Source/WebGPU/WGSL/Lexer.cpp

```diff
@@ -424,6 +463,11 @@ FOREACH_KEYWORD(MAPPING_ENTRY)
             if (UNLIKELY(reservedWordSet.contains(view)))
                 return makeToken(TokenType::ReservedWord);
 
+
+            if (UNLIKELY(length >= 2 && *startOfToken == '_' && *(startOfToken + 1) == '_'))
```

Comment:
In JavaScript, we also allow `$`. Does WGSL allow it?

Reply:
I don't know if `XID_Start` includes it, but the spec only specifies `XID_Start` + `_`: https://www.w3.org/TR/WGSL/#identifiers